sdk/src/main/java/com/mapbox/maps/MapController.kt

```diff
@@ -56,7 +56,10 @@ internal class MapController : MapPluginProviderDelegate, MapControllable {
   ) {
     this.renderer = renderer
     this.mapInitOptions = mapInitOptions
-    AssetManagerProvider().initialize(mapInitOptions.context.assets)
+    if (assetsManagerNeedInit) {
```

Comment:
shall we keep the assetManager static instead of using a static boolean?

Reply:
like using a static initialiser

Reply:
wouldn't we leek context in this case?